examples/complete/main.tf

```diff
@@ -30,6 +30,8 @@ module "db" {
   engine_version    = "5.7.11"
   instance_class    = "db.t2.large"
   allocated_storage = 5
+  storage_encrypted = true
```

Comment:
Let's turn encryption off in this example because it will create KMS key which we don't manage anyhow and it won't be destroyed by `terraform destroy`.

Reply:
Yeah, so true.